nomos-services/consensus/src/lib.rs

```diff
@@ -229,7 +229,8 @@ where
                 )
                 .await
                 else {
-                    unreachable!()
+                    tracing::debug!("Failed to gather initial votes");
+                    return Event::None;
```

Comment:
@Zeegomo This should be ok right? I have been working with Gusto and in reality if failing to gather the initial votes nodes should catch up anyway.

Reply:
Failing to gather initial votes means we go to the timeout routine, why does this happen now?